pmoves/services/yt-cookie-writer/main.py

```
@@ -0,0 +1,168 @@
+    except Exception:
+        return None
+
+
+def _decrypt(value: str) -> str:
+    """Decrypt Fernet-encrypted value, or return as-is."""
+    f = _get_fernet()
+    if f is None or not value:
+        return value
+    try:
+        return f.decrypt(value.encode()).decode()
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)